docs/quickstart/20_setup_flowehr_locally.qmd

```diff
@@ -4,7 +4,20 @@ title: Set-up & deployment
 
 This guide will take you through the basic set up and deployment for FlowEHR in your environment.
 
-## Forking the repo
+## Get the repositories
+
+Repositories:
+1. [FlowEHR-Data-Sources](https://github.com/UCLH-Foundry/FlowEHR-Data-Sources) - fork or clone this repo.
+2. [FlowEHR](https://github.com/UCLH-Foundry/FlowEHR) - this a template repo so you create your own repo from the template and customise it to your institution.
+3. [VM-Image-Gallery](https://github.com/UCLH-Foundry/VM-Image-Gallery) - this is a private repo
+4. [UCLH-Azure-TRE](https://github.com/UCLH-Foundry/FlowEHR-AzureTRE)  - this is a private repo
+
+Additional Repositories used:
+1. [FlowEHR-Data-Sources](https://github.com/UCLH-Foundry/FlowEHR-Data-Sources) - uses [Inform-DB](https://github.com/inform-health-informatics/Inform-DB/) and [Satellite](https://github.com/UCLH-Foundry/Satellite)
+2. [FlowEHR](https://github.com/UCLH-Foundry/FlowEHR) - uses [Data-Forge](https://github.com/UCLH-Foundry/FlowEHR-Data-Forge.git)
+3. [Powershell-Scripts](https://github.com/UCLH-Foundry/Powershell-Scripts) - allows VM Customisations of windows VMs
```

Comment:
This is public documentation so should probably not have any of the private repos, but instead say that you should create your own from the templates and not pointing towards our private repos